docs/src/api.md

```diff
@@ -100,6 +100,64 @@ ADC
 Delay
 ```
 
+## Functions for structs
+
+### `dur`
+```@docs
+dur
+```
+
+### `rotx`
+```@docs
+rotx
+```
+
+### `roty`
+```@docs
+roty
+```
+
+### `rotz`
+```@docs
+rotz
+```
+
+### `get_kspace`
+```@docs
+get_kspace
+```
+
+### `get_M1`
+```@docs
+get_M1
+```
+
+### `get_M2`
+```@docs
+get_M2
+```
+
+### `get_flip_angles`
+```@docs
+get_flip_angles
+```
+
+### `is_RF_on`
+```@docs
+is_RF_on
+```
+
+### `is_GR_on`
+```@docs
+is_GR_on
+```
+
+### `is_ADC_on`
+```@docs
+is_ADC_on
+```
+
+
 ## Read Data
```

Comment:
Not sure what this section mean, but KomaMRIFiles.read_phantom_jemris is missing, and others.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/commit/109eb655328e48204bc025bc331484464822a94a. Renamed to "Input/Output.